### PR TITLE
Rebuilt the minified files and added a userTimings example.

### PR DESCRIPTION
**Pull request by Emeegeemee**

### CODE REVIEW

File: example/index.html [hunk 1]
```diff
@@ -55,15 +57,27 @@
...
 	})
-	.controller('SampleCtrl', function () {});
+	.controller('ExampleCtrl', function($timeout, $analytics) {
+		this.userTimingsTest = function() {
+			var start = Date.now();
+			$timeout(function() {
+				var props = {
+					timingCategory: 'Testing',
+					timingVar: 'UserTimings',
+					timingValue:  Date.now() - start
```
Comment (ross-nordstrom) [inline example/index.html]: Why not abstract out the `timing` prefix, just like was done for Events.

Yes, they prefix it in the Analytics docs, but that's not the established usage in Angulartics

2016-01-13-094028_690x73_scrot
2016-01-13-094021_703x64_scrot

**Comment by maliknur** [comment]
Any update on this pull request?

**Comment by timelf123** [comment]
@Emeegeemee any chance you could take a look at this again?

**Comment by Emeegeemee** [comment]
Sorry for the silence. It looks like I don't have the branch anymore so I sent pr #61 which did't need to rebuild the binary.

**Comment by timelf123** [comment]
Thanks @Emeegeemee it was merged

**Pull request closed without merging**